src/components/stickyFooter/StickyFooter.scss

```diff
@@ -0,0 +1,45 @@
+@import '~coveo-styleguide/scss/common/palette.scss';
+@import '~coveo-styleguide/scss/variables.scss';
+
+$stickyFooterHeight: 58px;
+
+:export { stickyFooterHeight: $stickyFooterHeight; }
+
+.stickyFooter {
+    position: fixed;
+    width: 100%;
+    bottom: 0;
+    right: 0;
+    align-items: center;
+    justify-content: flex-end;
+  
+    display: flex;
+    overflow: hidden;
+  
+    box-shadow: 0 -1px 5px 0 rgba(0,0,0,0.3);
+    background-color: $white;
+    border-radius: 0 0 $border-radius $border-radius;
+    padding: $modal-footer-small-padding;
+
+
+    margin-bottom: -$navigation-width;
+
+    transform: translate3d(0, 0, 0);
+
+    transition: margin-bottom $navigation-toggle-duration ease-in-out, width $navigation-toggle-duration ease-in-out;
+    will-change: transform;
```

Comment:
Is this whole style to handle the side nav really necessary? I think I removed a bunch of it for the side nav to prevent problem in the admin

Reply:
i honestly kind of copied what was in vapor, but it's to handle the side nav and the display/hide of the footer, there are two cases to handle